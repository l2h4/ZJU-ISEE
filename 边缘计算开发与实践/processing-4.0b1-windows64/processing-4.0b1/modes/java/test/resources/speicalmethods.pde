ArrayList<Integer> positions = new ArrayList<>();

void setup() {
    size(100, 100);
    positions.add(25);
    positions.add(50);
    positions.add(75);
}

void draw() {
    for (int i = 0; i < positions.size(); i++) {
        ellipse(positions.get(i), positions.get(i), 10, 10);
    }
}

void mousePressed() {
    positions.add(mouseX);
}
