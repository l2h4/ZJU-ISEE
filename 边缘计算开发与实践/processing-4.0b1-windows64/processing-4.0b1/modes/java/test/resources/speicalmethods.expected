import processing.core.*;
import processing.data.*;
import processing.event.*;
import processing.opengl.*;

import java.util.HashMap;
import java.util.ArrayList;
import java.io.File;
import java.io.BufferedReader;
import java.io.PrintWriter;
import java.io.InputStream;
import java.io.OutputStream;
import java.io.IOException;

public class speicalmethods extends PApplet {

ArrayList<Integer> positions = new ArrayList<>();

public void setup() {
    /* size commented out by preprocessor */;
    positions.add(25);
    positions.add(50);
    positions.add(75);
}

public void draw() {
    for (int i = 0; i < positions.size(); i++) {
        ellipse(positions.get(i), positions.get(i), 10, 10);
    }
}

public void mousePressed() {
    positions.add(mouseX);
}


    public void settings() { size(100,100); }

    static public void main(String[] passedArgs) {
        String[] appletArgs = new String[] { "speicalmethods" };
        if (passedArgs != null) {
            PApplet.main(concat(appletArgs, passedArgs));
        } else {
            PApplet.main(appletArgs);
        }
    }
}